libs/ng-strapi-auth/src/lib/core/interceptor/http.auth-interceptor.ts

```diff
@@ -0,0 +1,27 @@
+
+import { HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http'
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+
+@Injectable()
+export class HttpAuthInterceptor implements HttpInterceptor {
+
+    constructor(private readonly route: Router) { }
+
+    intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+        const token = sessionStorage.get('currentJwt');
```

Comment:
Maybe we could have a localStorageKey enum to be sure we share the same variables there